discord_bots/cogs/in_progress_game.py

```diff
@@ -591,14 +598,17 @@ def update_ratings(
                 # stale
                 player.rated_trueskill_mu = trueskill_rating.mu
                 player.rated_trueskill_sigma = trueskill_rating.sigma
+
+                # There's already an appropriate category/position trueskill so
+                # just update it
                 if player.id in player_category_trueskills_by_id:
                     pct = player_category_trueskills_by_id[player.id]
                     pct.mu = trueskill_rating.mu
                     pct.sigma = trueskill_rating.sigma
                     pct.rank = trueskill_rating.mu - 3 * trueskill_rating.sigma
                     pct.last_game_finished_at = game_finished_at
-                    pct.position_id = team_gip.position_id
```

Comment:
this was the second bug, it was overwriting whatever the position was on the trueskill